### PR TITLE
fix vue warnings introduced by new props handling of LKSG form fields

### PR DESCRIPTION
**Pull request by janherre**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one E2E Test exists testing the new feature
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] The new version is deployed with fake data to the dev server "dev1" using this branch
  - [x] Run with setting `Use real data` turned off and `Reset backend database & Re-populate` turned on 
  - [x] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [x] It's verified that the CD run is green
  - [x] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [x] The new version is deployed with real data to a dev server using this branch
  - [x] Run with both settings `Reset backend database & Re-populate` and `Use real data` turned on
  - [x] It's verified that the CD run is green
- [x] If any work on the UI is to be merged, those changes were also documented in the Figma

### CODE REVIEW

File: dataland-frontend/src/components/forms/parts/fields/FormFieldProps.ts [hunk 1]
```diff
@@ -27,28 +27,40 @@ export const FormFieldProps = {
   },
 };
 
-export const FormFieldPropsWithPlaceholder = Object.assign(FormFieldProps, {
-  required: {
+export const YesNoFormFieldProps = Object.assign(JSON.parse(JSON.stringify(FormFieldProps)) as object, {
```
[the remark this answers is not in the document]
Comment (janherre) [reply dataland-frontend/src/components/forms/parts/fields/FormFieldProps.ts:30]: Das wird erst im upload documents for LkSG branch eingesetzt, ist wohl im Commit verloren gegangen. Ich kann das auch noch in diesem PR mit committen damit es klar ist.

[the remark this answers is not in the document]
Comment (SiegfriedSobkowiak) [reply dataland-frontend/src/components/forms/parts/fields/FormFieldProps.ts:30]: Dann würde ich es eher hier rausnehmen. Das kommt dann ja mit dem anderen Branch.

[the remark this answers is not in the document]
Comment (janherre) [reply dataland-frontend/src/components/forms/parts/fields/FormFieldProps.ts]: Ist raus

File: dataland-frontend/src/components/forms/parts/fields/FormFieldProps.ts [hunk 2]
```diff
@@ -27,28 +27,40 @@ export const FormFieldProps = {
...
+  },
+});
+export const FormFieldPropsWithPlaceholder = Object.assign(JSON.parse(JSON.stringify(FormFieldProps)) as object, {
+  placeholder: {
     type: String,
     default: "",
   },
 });
-export const DateFormFieldProps = Object.assign(FormFieldPropsWithPlaceholder, {
+export const DateFormFieldProps = Object.assign(JSON.parse(JSON.stringify(FormFieldPropsWithPlaceholder)) as object, {
```
[the remark this answers is not in the document]
Comment (janherre) [reply dataland-frontend/src/components/forms/parts/fields/FormFieldProps.ts]: es gibt jetzt eine `deepCopyObject(obj)` funktion in den updateObjectUtils

**Pull request merged**